### ISSUE TITLE
ClassGraph doesn't scan Application ClassLoader when provided as override on JDK11

### ISSUE THREAD

**Issue by chrisr3** [classgraph/classgraph#411]
This problem looks like an edge-case of #382 . We are using ClassGraph 4.8.58, but the problem still exists with 4.8.65.

Basically, we are configuring ClassGraph with
```kotlin
ClassGraph().overrideClassLoaders(classloader)
    .ignoreParentClassLoaders()
    .enableClassInfo()
    .scan()
```
The `classloader` parameter is usually expected to be an instance of `URLClassLoader` but in some circumstances it may be the application classloader, and Java 8 is fine with this. However, ClassGraph doesn't find anything inside the application classloader on Java 11! The fix from #382 does enable module scanning, but the "Module Finder" still doesn't find any modules to scan.

We are currently working around this using:
```kotlin
ClassGraph().apply {
    if (classloader !== ClassLoader.getSystemClassLoader()) {
        overrideClassLoaders(classloader)
    }
}.ignoreParentClassLoaders()
 .enableClassInfo()
 .scan()
}

**Comment by lukehutch** [comment]
You're right -- the fix to #382 was incomplete. Thanks for reporting.

Fixed in git master. Will be released in 4.8.67, although currently Sonatype is having issues with pushing out releases, so it may take a few hours or days for 4.8.67 to hit Maven Central.

**Comment by lukehutch** [comment]
I found a workaround for the Sonatype issue, so the fix is now published in classgraph-4.8.67.

**Comment by lukehutch** [comment]
Properly fixed now in 4.8.68.
